packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -213,10 +231,10 @@ export default class Meeting extends StatelessSparkPlugin {
   setRemoteStreams(pc) {
     // Receiving remote media streams
     pc.ontrack = (event) => {
-      [this.mediaStream, this.localMediaStream] = event.streams;
-
-      this.trigger('media:ready', {
-        type: 'remote',
+      [this.mediaStream] = event.streams;
+      document.getElementById('remotevideo').srcObject = this.mediaStream;
```

Comment:
Please remove. This isn't necessary if we handle the events in the app itself as a developer would

Reply:
?? lets discuss tomorrow during the sanity check -- want to have the one that got set up for the guest client

Reply:
Sure check `/samples/browser-meetings/app.js` line 118 or so. It listens for the event and does the same thing you are doing on 235